logging/logging.gradle

```diff
@@ -56,6 +56,7 @@ sourceSets {
     }
 }
 
+findbugs.sourceSets = [sourceSets.main]
```

Comment:
findbugs "found a bug" in the tests. It doesn't like that the watcher is public and never used. Since that's not how it works and I don't want to spend time making a filter file for the tests right now, we'll only run findbugs on shipping code.

Reply:
Makes sense!